### DISCUSSION TITLE
How to add a prompt in cyrillic?

### DISCUSSION THREAD

**Discussion by EvKhristoforova** [jspsych/jsPsych#1450] [Q&A] [answered]
As simple as that. If you specify a prompt for a trial in Russian, in the Chrome you get a mess "Ð¢Ð°Ðº Ð¼Ð¾Ð¶Ð½Ð¾ Ð½Ð". Is there any cure for that?

**Comment by jodeleeuw** [comment, marked as answer]
Hi @EvKhristoforova,

The trick is to specify the encoding of the document in a `<meta>` tag. Check out [this discussion thread](https://github.com/jspsych/jsPsych/discussions/813).

**Reply by EvKhristoforova** [reply]
It works! Thanks a lot!
